Live TV (hdhomerun) Stopped working after upgrade to 10.3.6.

**Describe the bug**
After in place upgrade from 10.3.5 to 10.3.6 hdhomerun no longer works for live TV. Reverting back to 10.3.5 and hdhomerun works again.

Ubuntu Bionic 64bit latest updates installed

Did some testing with @dinki regarding the changes from #1473. It isn't

```
         public string DownloadFilePath { get; set; }
```

And trying to readd

```
        public void SetPostData(IDictionary<string, string> values)
        {
            var strings = values.Keys.Select(key => string.Format("{0}={1}", key, values[key]));
            var postContent = string.Join("&", strings.ToArray());

            RequestContent = postContent;
            RequestContentType = "application/x-www-form-urlencoded";
        }
```

Caused a build failure.

```
  Restore completed in 25.36 ms for /repo/RSSDP/RSSDP.csproj.
  Restore completed in 15.52 sec for /repo/Jellyfin.Drawing.Skia/Jellyfin.Drawing.Skia.csproj.
  Failed to download package 'runtime.linux-x64.Microsoft.NETCore.DotNetHostResolver.2.1.9' from 'https://api.nuget.org/v3-flatcontainer/runtime.linux-x64.microsoft.netcore.dotnethostresolver/2.1.9/runtime.linux-x64.microsoft.netcore.dotnethostresolver.2.1.9.nupkg'.
  The download of 'https://api.nuget.org/v3-flatcontainer/runtime.linux-x64.microsoft.netcore.dotnethostresolver/2.1.9/runtime.linux-x64.microsoft.netcore.dotnethostresolver.2.1.9.nupkg' timed out because no data was received for 60000ms.
    Exception of type 'System.TimeoutException' was thrown.
  Restore completed in 1.29 min for /repo/Jellyfin.Server/Jellyfin.Server.csproj.
  Emby.XmlTv -> /repo/Emby.XmlTv/Emby.XmlTv/bin/Release/netstandard2.0/Emby.XmlTv.dll
  MediaBrowser.Model -> /repo/MediaBrowser.Model/bin/Release/netstandard2.0/MediaBrowser.Model.dll
Net/HttpRequestOptions.cs(116,39): error CS1061: 'ICollection<string>' does not contain a definition for 'Select' and no accessible extension method 'Select' accepting a first argument of type 'ICollection<string>' could be found (are you missing a using directive or an assembly reference?) [/repo/MediaBrowser.Common/MediaBrowser.Common.csproj]
  DvdLib -> /repo/DvdLib/bin/Release/netstandard2.0/DvdLib.dll
  BDInfo -> /repo/BDInfo/bin/Release/netstandard2.0/BDInfo.dll
The command '/bin/sh -c bash -c "source deployment/common.build.sh &&     build_jellyfin Jellyfin.Server Release linux-x64 /jellyfin"' returned a non-zero code: 1
```

Those were the only two changes that were suspicious to me, so it might indeed be that second function (which needs more investigating on the failure), or it's not a change in `MediaBrowser.Common/Net/HttpRequestOptions.cs` that's to blame.

Any other input helpful!

--- a/Emby.Server.Implementations/HttpClientManager/HttpClientManager.cs
+++ b/Emby.Server.Implementations/HttpClientManager/HttpClientManager.cs
@@ -284,7 +284,7 @@ namespace Emby.Server.Implementations.HttpClientManager
 
             if (!options.BufferContent)
             {
-                var response = await client.SendAsync(httpWebRequest, options.CancellationToken).ConfigureAwait(false);
+                var response = await client.SendAsync(httpWebRequest, HttpCompletionOption.ResponseHeadersRead, options.CancellationToken).ConfigureAwait(false);
 
                 await EnsureSuccessStatusCode(response, options).ConfigureAwait(false);
 
@@ -296,7 +296,7 @@ namespace Emby.Server.Implementations.HttpClientManager
                     Content = stream,
                     StatusCode = response.StatusCode,
                     ContentType = response.Content.Headers.ContentType?.MediaType,
-                    ContentLength = stream.Length,
+                    ContentLength = response.Content.Headers.ContentLength,
                     ResponseUrl = response.Content.Headers.ContentLocation?.ToString()
                 };
             }

--- a/Emby.Server.Implementations/HttpClientManager/HttpClientManager.cs
+++ b/Emby.Server.Implementations/HttpClientManager/HttpClientManager.cs
@@ -301,7 +301,7 @@ namespace Emby.Server.Implementations.HttpClientManager
                 };
             }
 
-            using (var response = await client.SendAsync(httpWebRequest, options.CancellationToken).ConfigureAwait(false))
+            using (var response = await client.SendAsync(httpWebRequest, HttpCompletionOption.ResponseHeadersRead, options.CancellationToken).ConfigureAwait(false))
             {
                 await EnsureSuccessStatusCode(response, options).ConfigureAwait(false);
 

